akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/MaxThroughputCOMPRESSEDSpec.scala

```diff
@@ -0,0 +1,284 @@
+/**
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.remote.artery
+
+import java.util.concurrent.Executors
+import java.util.concurrent.TimeUnit.NANOSECONDS
+import akka.remote.artery.compress.CompressionSettings
+
+import scala.concurrent.duration._
+import akka.actor._
+import akka.remote.RemoteActorRefProvider
+import akka.remote.testconductor.RoleName
+import akka.remote.testkit.MultiNodeConfig
+import akka.remote.testkit.MultiNodeSpec
+import akka.remote.testkit.STMultiNodeSpec
+import akka.testkit._
+import com.typesafe.config.ConfigFactory
+import java.net.InetAddress
+
+object MaxThroughputCOMPRESSEDSpec extends MultiNodeConfig {
```

Comment:
Yes, I'll remove now since compression is now usable without breaking anything else.